docs/setup-and-installing.md

```diff
@@ -2,151 +2,152 @@
 
 -----
 
-# Set up and get started with the SpatialOS GDK for Unity
```

Comment:
should we keep the header?

Reply:
Yes Laura commented on this in the doc - was a mistake